tests/codebase/test_python_execution_with_OO.py

```diff
@@ -31,6 +31,8 @@
 # Tests
 #-----------------------------------------------------------------------------
 
+blacklist = {"bokeh.embed.notebook.widgets"}
```

Comment:
suggest making an issue to conditionally remove from blacklist if the ipywigets version is high enough and leaving a comment here pointing at the issue